### DISCUSSION TITLE
security : data stored inside the html

### DISCUSSION THREAD

**Discussion by FlorianJ0** [Data-Centric-AI-Community/fg-data-profiling#998] [Q&A] [answered]
Hi,
I am working on sensitive data I was wondering if the complete data used to generate the plots and reports are saved inside the html or not ?
Best,
Florian

**Comment by sbrugman** [comment, marked as answer]
The data are not saved inside the HTML. There may be records shown in the samples or duplicates section. Please see the documentation for more information: https://pandas-profiling.ydata.ai/docs/master/rtd/pages/sensitive_data.html

**Reply by FlorianJ0** [reply]
Damn, sorry I missed the documentation. Thank you for the answer !
